### ISSUE TITLE
F10 does not step into the entry point of an executable with mainCRTStartup() or WinMainCRTStartup()

### ISSUE THREAD

**Issue by zobermiller** [EpicGames/raddebugger#61]
If I load an executable target into the debugger that has been built without the CRT `main()` or `WinMain()` entry points and instead was built with `mainCRTStartup()` or `WinMainCRTStartup()`, hitting F10 does not place me at the beginning of my entry point but instead somewhere inside ntdll.dll.

Here's a sample application:

```
#include <stdio.h>

int mainCRTStartup()
{
    int x = 5;
    int y = 10;

    printf("%d + %d = %d\n", x, y, x + y);
    return 0;
}
```

and build it with

```
cl /nologo /O2 /Zi /W3 /WX /GS- main.c /link /fixed /incremental:no /opt:icf /opt:ref /subsystem:console ucrt.lib
```

If I replace the `mainCRTStartup()` with `main()` and run the same command (excluding the `ucrt.lib` at the end), F10 will step directly into the beginning of `main()`.

**Comment by ryanfleury** [comment]
`WinMainCRTStartup` should work, but the debugger was not previously looking for `mainCRTStartup`. Can you try pulling 25c31b0d7782ab36b9c15c1069e8a7c1eb390919, building, and running again?

**Comment by zobermiller** [comment]
Yes, that worked perfectly. Thank you, Ryan! And I'm sorry about the wrong information regarding `WinMainCRTStartup()`. I must've been testing something else and misconstrued it as that instead.
